.github/workflows/ci.yml

```diff
@@ -147,15 +147,15 @@ jobs:
       run: cd plrustc && ./build.sh && cp ../build/bin/plrustc ~/.cargo/bin
 
     - name: Run cargo pgx init for PG14
```

Comment:
update `- name:`, I suppose now that it's parameterized.

Reply:
Yeah good point.